csharp/Worker/Microsoft.Spark.CSharp/Worker.csproj

```diff
@@ -58,9 +58,7 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
-    <None Include="App.config">
```

Comment:
The APP.config is always be copied without the change. Actually, APP.config always be copied and renamed to <application>.exe.config by default. So this change is for not coping App.config and only keep CSharpWorker.exe.config.